parts/header.html

```diff
@@ -1,7 +1,7 @@
 <!-- wp:group {"layout":{"inherit":"true"}} -->
-<div class="wp-block-group">
-	<!-- wp:group {"align":"full","layout":{"type":"flex","justifyContent":"space-between"},"style":{"spacing":{"padding":{"bottom":"var(--wp--preset--spacing--50)","top":"var(--wp--preset--spacing--50)"}}}} -->
-	<div class="wp-block-group alignfull" style="padding-bottom:var(--wp--preset--spacing--50);padding-top:var(--wp--preset--spacing--50)">
+<div class="wp-block-group" style="display: flex;flex-direction: row;justify-content: center;">
+	<!-- wp:group {"align":"full","layout":{"type":"flex","justifyContent":"space-between"},"style":{"spacing":{"padding":{"bottom":"3.75rem","top":"3.75rem","width":"1000px"}}}} -->
```

Comment:
Same comment here as with the footer. Did you consider using CSS variables as the base theme does? Otherwise, if that value changes, we have to change it everywhere, which kinda defeats the purpose of having a `theme.json` file.

Reply:
same